torch/nn/functional.py

```diff
@@ -38,7 +38,8 @@ def conv2d(input, weight, bias=None, stride=1, padding=0, dilation=1,
         groups: split input into groups, in_channels should be divisible by
           the number of groups
 
-    Examples:
+    Examples::
```

Comment:
This has already been updated [here](https://github.com/pytorch/pytorch/pull/2248/files).